### ISSUE TITLE
gktorrent and oxtorrent - Error downloading 

### ISSUE THREAD

**Issue by bibi32** [Jackett/Jackett#6398]
Jackett v0.12.1007 LinuxARM64
Hello, I have an error with gktorrent and oxtorrent, I can't download torrent when I found it.
Could you watch this please.

Error downloading to blackhole gktorrent
Error | CardigannIndexer (gktorrent): Download selector a[href^="/download/"] didn't match:
[jackett.log](https://github.com/Jackett/Jackett/files/3865048/jackett.log)
[jackett2.log](https://github.com/Jackett/Jackett/files/3865071/jackett2.log)

**Comment by garfield69** [comment]
gktorrent have changed their download link. 
when I browse the site directly, and visit the details page of any entry from the search page,
I see two buttons: the **Telecharger** button is now a dummy button that always takes you to click bait sites
and the **Magnet** button is the one that has an actual magnet link to download.

so I have changed to the gktorrent indexer to switch to using the magnet link for downloads.

oxtorrent are doing something similar, the first time you click on the **Telecharger** button you get a redirect to click bait, the second time it downloads.
but the **Magnet** is always available so I've switched the indexder to using the magnet here as well.

**Comment by bibi32** [comment]
Thanks

**Comment by garfield69** [comment]
try out jackett 0.12.1014

**Comment by bibi32** [comment]
the .magnet is good but rutorrent doesn't recognize it.
I need to search a solution.

**Comment by bibi32** [comment]
Hello, I can't download with medusa and radarr with Torrent9_clone, Oxtorrent and Gktorrent.
With medusa I have a result but it would'nt download and on radarr I have no result in my search.

**Comment by garfield69** [comment]
unfortunately it is no longer possible to download from these 3 sites via indexers written in yaml for the cardigann processor.

the cardigann processor expects to be able to use the **href** attribute of an element but these 3 sites now load the href with `javascript:` and use the **onclick** attribute to store the url
```html
<a class='btn btn-danger download' href='javascript:' onclick=location.href='magnet:?xt=urn:btih:YDKLK6U6KCO6YFZQ27MANKGTBN5V5NX2&tr=udp://tracker.opentrackr.org:1337/announce&tr=udp://9.rarbg.me:2710/announce&tr=udp://9.rarbg.to:2740/announce&tr=udp://9.rarbg.com:2770/announce&tr=udp://explodie.org:6969/announce&tr=udp://tracker.internetwarriors.net:1337/announce&tr=udp://ipv4.tracker.harry.lu:80/announce&tr=udp://tracker.tiny-vps.com:6969/announce&tr=http://servandroidkino.ru/announce&tr=http://1337.abcvg.info/announce&tr=udp://open.stealth.si:80/announce&tr=http://tracker.bittor.pw:1337/announce&tr=http://tracker3.itzmx.com:8080/announce&tr=http://open.acgnxtracker.com/announce&tr=http://share.camoe.cn:8080/announce&tr=udp://bt.xxx-tracker.com:2710/announce&tr=udp://exodus.desync.com:6969/announce&tr=udp://open.demonii.si:1337/announce&tr=udp://tracker.torrent.eu.org:451/announce&tr=udp://tracker.cyberia.is:6969/announce&tr=udp://151.80.120.114:2710/announce&tr=http://91.217.91.21:3218/announce&tr=http://tracker2.itzmx.com:6961/announce&tr=udp://tracker.yoshi210.com:6969/announce&tr=http://tracker.yoshi210.com:6969/announce&tr=http://tracker.tvunderground.org.ru:3218/announce&tr=http://torrentsmd.com:8080/announce&tr=udp://amigacity.xyz:6969/announce&tr=udp://newtoncity.org:6969/announce&tr=udp://zephir.monocul.us:6969/announce&tr=udp://denis.stalker.upeer.me:6969/announce&tr=http://tracker.bt4g.com:2095/announce&tr=http://open.trackerlist.xyz:80/announce&tr=udp://opentor.org:2710/announce' style='background: rgba(0, 0, 0, 0) linear-gradient(to bottom, rgba(254, 48, 0, 0.8) 0%, rgba(172, 20, 0, 0.8) 100%) repeat scroll 0 0;height:36px;line-height: 33px;padding-top: 0px;'><i class='fa fa-magnet'></i> Lien Magnet</a>
```
untill a c# dev volunteer can amend the cardigann processor to handle this variation then jackett will not be able to support downloads from these 3 sites.
ping to any c# volunteers to help with this one.

**Comment by bibi32** [comment]
Ok ...

**Comment by ngosang** [comment]
@garfield69 I think this can be fixed in Cardigann after #6891

**Comment by garfield69** [comment]
looks like gktorrent and oxtorrent have changed their download links back to normal again.
I have no problems using the current indexers to fetch either .torrent or magnet (depending on the config settings)
